src/transformations/simplify.jl

```diff
@@ -389,9 +386,29 @@ function _flat_tolerances(f, points)
     result[1] = result[end] = Inf
 
     for i in 2:length(result) - 1
-        result[i] = f(points[i-1], points[i], points[i+1])
+        result[i] = f(points[i], points[i-1], points[i+1])
     end
     return result
 end
 
 _remove!(s, i) = s[i:end-1] .= s[i+1:end]
+
```

Comment:
So in the future would you prefer that this be a new PR?

Reply:
Thanks that would be nice! But I also posted ColPrac to distance this request from being just a personal preference. See:

    PRs that move code should not also change code, so that they are easier to review.

        If only moving code, review for correctness is not required.

        If only changing code, then the diff makes it clear what lines have changed.

    PRs with large improvements to style should not also change functionality.

        This is to avoid making large diffs that are not the focus of the PR.
        While it is often helpful to fix a few typos in comments on the way past, it is different from using a regex or formatter on the whole project to fix spacing around operators.

Following this also makes it easier to point out my  own bad coding practices